oaknational/Oak-Web-Application
# Chore/remove old legacy api code

## Description

Music year: 2008

- Removes all references to old API
- Moves schema and fixtures to new API

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2371--oak-web-application.netlify.thenational.academy
Everything should work as before (sorry Jack)

## Screenshots


## Checklist
